include/temoto_robot_manager/robot.h

```diff
@@ -24,6 +24,7 @@
 #include "temoto_robot_manager/robot_common_procedures.h"
 #include "temoto_robot_manager/GripperControl.h"
 #include "temoto_robot_manager/custom_plugin_helper.h"
+#include "temoto_robot_manager/navigation_plugin_helper.h"
 #include <moveit/move_group_interface/move_group_interface.h>
 #include <moveit/planning_interface/planning_interface.h>
 #include <move_base_msgs/MoveBaseAction.h>
```

Comment:
Not needed anymore.

Reply:
removed the #include <move_base_msgs/MoveBaseAction.h>